src/handlers/google.ts

```diff
@@ -81,6 +81,383 @@ export default async function google(
+  "www.google.com.do",
+  "www.google.dz",
+  "www.google.com.ec",
+  "www.google.ee",
+  "www.google.com.eg",
+  "www.google.es",
+  "www.google.com.et",
+  "www.google.fi",
+  "www.google.com.fj",
+  "www.google.fm",
+  "www.google.fr",
+  "www.google.ga",
+  "www.google.ge",
+  "www.google.gg",
+  "www.google.com.gh",
+  "www.google.com.gi",
+  "www.google.gl",
+  "www.google.gm",
+  "www.google.gr",
+  "www.google.com.gt",
+  "www.google.gy",
+  "www.google.com.hk",
+  "www.google.hn",
+  "www.google.hr",
+  "www.google.ht",
+  "www.google.hu",
+  "www.google.co.id",
+  "www.google.ie",
+  "www.google.co.il",
+  "www.google.im",
+  "www.google.co.in",
+  "www.google.iq",
+  "www.google.is",
+  "www.google.it",
+  "www.google.je",
+  "www.google.com.jm",
+  "www.google.jo",
+  "www.google.co.jp",
+  "www.google.co.ke",
+  "www.google.com.kh",
+  "www.google.ki",
+  "www.google.kg",
+  "www.google.co.kr",
+  "www.google.com.kw",
+  "www.google.kz",
+  "www.google.la",
+  "www.google.com.lb",
+  "www.google.li",
+  "www.google.lk",
+  "www.google.co.ls",
+  "www.google.lt",
+  "www.google.lu",
+  "www.google.lv",
+  "www.google.com.ly",
+  "www.google.co.ma",
+  "www.google.md",
+  "www.google.me",
+  "www.google.mg",
+  "www.google.mk",
+  "www.google.ml",
+  "www.google.com.mm",
+  "www.google.mn",
+  "www.google.com.mt",
+  "www.google.mu",
+  "www.google.mv",
+  "www.google.mw",
+  "www.google.com.mx",
+  "www.google.com.my",
+  "www.google.co.mz",
+  "www.google.com.na",
+  "www.google.com.ng",
+  "www.google.com.ni",
+  "www.google.ne",
+  "www.google.nl",
+  "www.google.no",
+  "www.google.com.np",
+  "www.google.nr",
+  "www.google.nu",
+  "www.google.co.nz",
+  "www.google.com.om",
+  "www.google.com.pa",
+  "www.google.com.pe",
+  "www.google.com.pg",
+  "www.google.com.ph",
+  "www.google.com.pk",
+  "www.google.pl",
+  "www.google.pn",
+  "www.google.com.pr",
+  "www.google.ps",
+  "www.google.pt",
+  "www.google.com.py",
+  "www.google.com.qa",
+  "www.google.ro",
+  "www.google.ru",
+  "www.google.rw",
+  "www.google.com.sa",
+  "www.google.com.sb",
+  "www.google.sc",
+  "www.google.se",
+  "www.google.com.sg",
+  "www.google.sh",
+  "www.google.si",
+  "www.google.sk",
+  "www.google.com.sl",
+  "www.google.sn",
+  "www.google.so",
+  "www.google.sm",
+  "www.google.sr",
+  "www.google.st",
+  "www.google.com.sv",
+  "www.google.td",
+  "www.google.tg",
+  "www.google.co.th",
+  "www.google.com.tj",
+  "www.google.tl",
+  "www.google.tm",
+  "www.google.tn",
+  "www.google.to",
+  "www.google.com.tr",
+  "www.google.tt",
+  "www.google.com.tw",
+  "www.google.co.tz",
+  "www.google.com.ua",
+  "www.google.co.ug",
+  "www.google.co.uk",
+  "www.google.com.uy",
+  "www.google.co.uz",
+  "www.google.com.vc",
+  "www.google.co.ve",
+  "www.google.co.vi",
+  "www.google.com.vn",
+  "www.google.vu",
+  "www.google.ws",
+  "www.google.rs",
+  "www.google.co.za",
+  "www.google.co.zm",
+  "www.google.co.zw",
+  "www.google.cat",
```

Comment:
The whole list should (not must, but should) be replaced with a several globs like in StackOverflow.